docs/building-a-form/available-components.md

```diff
@@ -175,7 +175,7 @@ The data for a group of radio buttons will be quite similar to the data for a se
 To override the `SelectWidget`, pass `'ui:widget': 'radio'` to your `uiSchema` for that field. To specify different label text for each option, pass `'ui:options'` to `uiSchema`.
```

Comment:
Well we have a table that gives an overview of all the widgets in adding-widgets-and-definitions.md, but we haven't added detailed documentation for all of the widgets. Not sure if it's really needed?

Reply:
Okay, yeah, it's mentioned [here](https://github.com/usds/us-forms-system/blob/master/docs/building-a-form/adding-widgets-and-definitions.md#common-widgets). I think that's fine. 👍

Reply:
Would it be helpful if I went through and linked all of those up around the docs where they're mentioned? At least the first mention in a topic.

Reply:
Do you mean include links in that table to other places in the docs? I think that would be helpful! If we don't have links for all of the widgets, though, do you think that would be a problem?

Reply:
I meant the other way around... I'd update the table to be section headers so that they could be linked to, and then anywhere else in the docs that mentions each widget, I'd add a link to it. Is that overkill?

Reply:
Ah gotcha. I think that works!

Reply:
Opened an issue for this in https://github.com/usds/us-forms-system/issues/178.